test/loopback-dl.test.js

```diff
@@ -698,7 +698,7 @@ describe('Load models with base', function() {
     Customer.definition.properties.name.should.have.property('type', String);
   });
 
-  it('should inherit properties by clone from base option', function() {
+  it('inherit properties by clone from base model', function() {
```

Comment:
`it('inherits...`

There are few more places below that don't use third-person singular and should be fixed too.